### ISSUE TITLE
Retinal shut vessels

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10906]
Preferred term label:
Retinal shut vessels

Synonyms
Retinal collateral vessels, Retinal shunt vessels

Definition (free text, please give PubMed ID)
Abnormal blood vessels in the retina that form as a response to vascular occlusion, serving as bypass channels to redirect blood flow around obstructed or damaged retinal vessels. Retinal shunt vessels are often associated with conditions like retinal vein occlusion or diabetic retinopathy and may appear as tortuous or dilated vessels on fundoscopic examination.

Parent term (use hpo.jax.org/app)
Abnormality of retinal vascular morphology (HP:8046)

Diseases characterized by this term
Retinal vein occlusion, Diabetic retinopathy

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01  -- note "shut" vessels seems to be a spelling error!

Adding new term: 
- ID: HP:6001183
- label: Retinal shunt vessels
- parents: [Abnormal retinal vascular morphology (HP:0008046)](https://hpo.jax.org/app/browse/term/HP:0008046)
- pmid: PMID:37238303
- def: bnormal blood vessels in the retina that form as a response to vascular occlusion, serving as bypass channels to redirect blood flow around obstructed or damaged retinal vessels. Retinal shunt vessels are often associated with conditions like retinal vein occlusion or diabetic retinopathy and may appear as tortuous or dilated vessels on fundoscopic examination.
- comment: See Figure 12 of PMID:37238303.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10906
- synonyms: Retinal collateral vessels(NONE)
